Carmu/Sources/Views/MyPage/MyPageView.swift

```diff
@@ -224,23 +231,24 @@ final class MyPageView: UIView {
             make.height.equalTo(340)
         }
         settingsButton.snp.makeConstraints { make in
-            make.top.equalTo(safeAreaLayoutGuide.snp.top).inset(12)
+            make.top.equalTo(safeAreaLayoutGuide).inset(12)
             make.trailing.equalToSuperview().inset(20)
         }
         imageView.snp.makeConstraints { make in
             make.width.height.equalTo(80)
-            make.trailing.equalTo(userInfoView.snp.trailing).inset(27)
-            make.bottom.equalTo(userInfoView.snp.bottom).inset(59)
+            make.trailing.equalToSuperview().inset(20)
+            make.bottom.equalToSuperview().inset(59)
         }
         addButton.snp.makeConstraints { make in
             make.trailing.bottom.equalTo(imageView)
+            make.width.height.equalTo(24)
         }
         nicknameStack.snp.makeConstraints { make in
             make.leading.equalToSuperview().inset(20)
-            make.bottom.equalTo(userInfoView).inset(62)
+            make.bottom.equalTo(userInfoView).offset(-67)
         }
```

Comment:
nicknameStack은 userInfoView의 subview니까 이렇게 쓰는게 좀 더 가독성이 좋을 것 같아요!

```suggestion
        nicknameStack.snp.makeConstraints { make in
            make.leading.equalToSuperview().inset(20)
            make.bottom.equalToSuperview().inset(67)
        }
```